bin/test-instruction-logger/bin/test-instruction-logger

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env php
+<?php
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+// Make sure this script is being run over the PHP CLI.
+if ( 'cli' !== php_sapi_name() ) {
+	return;
+}
+
+define('WP_USE_THEMES', false);
+
+$wc_admin_root = __DIR__ . '/../../../';
+require_once $wc_admin_root . '/../../../wp-load.php';
```

Comment:
@moon0326 Awesome, this is working for me now, but we just need to delete this line. 

```suggestion
```

It's testing fantastic 👍

Reply:
Thank you @adrianduffell 👍 

Removed it in https://github.com/woocommerce/woocommerce-admin/pull/7499/commits/05286fc8f73527164b30cb4f22bb8a3810375920